### PR TITLE
DV3D fixes

### PR DESCRIPTION
**Pull request by remram44**
Did this package ever work at all? It is full of errors according to my IDE, missing symbols, typos...

Took a look after UV-CDAT/uvcdat#1056. Marked the remaining errors from my IDE with `# RR0224:` since apparently the author didn't use one.

I have no idea what I'm doing, but this looks like it should either be massively reworked or completely removed from the codebase (to avoid more problems like 1056).

### CODE REVIEW

**Comment by ThomasMaxwell** [comment]
This code has been obsolete for nearly a year, the only reason it still remains in UVCDAT is because it contains code for the OpenDAP browser which is still in use.

**Comment by remram44** [comment]
Can't that be pulled out?

**Comment by ThomasMaxwell** [comment]
Yes it can be pulled out but we've all had more pressing issues to work on.  Please feel free to submit an issue report.

File: vistrails/packages/vtDV3D/API.py [hunk 1]
```diff
@@ -292,8 +292,8 @@ def run(self):
         gui.application.stop_application()        
         
 if __name__ == '__main__':
+    import cdutil, genutil
     cdmsfile = cdms2.open('~/data/AConaty/comp-ECMWF/ecmwf.xml')
```
Comment (doutriaux1) [inline vistrails/packages/vtDV3D/API.py:296]: hard coded path?

**Comment by doutriaux1** [comment]
@ThomasMaxwell I will let you review and approve this one. You know better what's going on here.

File: vistrails/packages/vtDV3D/API.py (hunk 1 above)
Comment (ThomasMaxwell) [inline vistrails/packages/vtDV3D/API.py:296]: This is debugging code embedded in obsolete code.   There should not be a need to review this code- it's not used anymore.

**Comment by ThomasMaxwell** [comment]
I don't know what problem this branch is supposed to be fixing.   The original issue, #1056, appears to be already fixed in uvcdat_master by the comment in front of line 23 in /vistrails/vistrails/packages/vtDV3D/InteractiveConfiguration.py".   I don't know what branch or commit introduced that change, but as far as I can tell it fixes the issue.

**Comment by doutriaux1** [comment]
@ThomasMaxwell yes #1056 is fixed by #14

**Comment by aashish24** [comment]
Relevant anymore or should we close it?

**Comment by ThomasMaxwell** [comment]
You can close it.

**Pull request closed without merging**